src/aura/STG_CMP_CourseConnections/STG_CMP_CourseConnections.cmp

```diff
@@ -172,4 +172,33 @@
       </div>
     </div>
   </div>
+
+  <!-- Course Description Data Migration -->
+  <div class="slds-card slds-m-top--medium">
+    <div class="slds-card__header slds-grid">
+      <header class="slds-media slds-media--center slds-has-flexi-truncate">
+        <div class="slds-media__body">
+          <h2>
+              <span class="slds-text-heading--small">Courses: Description Data Migration</span>
+          </h2>
+        </div>
+      </header>
+    </div>
+    <div class="slds-card__body">
+      <div class="slds-p-around--medium" style="background-color: #ffffff;">
+        <!-- Card Body -->
+        In HEDA 1.30 a new extended field was added to accommodate longer Course descriptions.<br />
```

Comment:
@lianeyee When you have a free minute, would you look over these instructions (there is also a screenshot I posted) to make sure they are clear.